packages/block-library/src/details/edit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export default ( { attributes, setAttributes } ) => {
+	return (
+		<>
+			<RichText
+				tagName="div"
+				value={ attributes.content }
+				onChange={ ( content ) => {
+					setAttributes( { content } );
+				} }
+				placeholder={ __( 'Visible Content' ) }
```

Comment:
The placeholder should be more descriptive, and this `RichText` input should probably also have an `aria-label` for screen reader users. Also, only the first letter should be capitalized.